network/soroban-rpc/rpc-providers.mdx

```diff
@@ -12,12 +12,14 @@ These providers allow access to the Testnet and Mainnet networks.
 | Provider                                                | Access                                              | Futurenet     | Testnet   | Mainnet     |
 | ------------------------------------------------------- | --------------------------------------------------- | ------------- | --------- | ----------- |
 | [Blockdaemon](https://www.blockdaemon.com)              | [Access](https://www.blockdaemon.com/apply/soroban) | Not available | Available | Available   |
-| [Validation Cloud](https://validationcloud.io/)         | [Access](https://app.validationcloud.io/)           | Not available | Available | Available   |
-| [QuickNode\*](https://www.quicknode.com/chains/stellar) | [Access](https://www.quicknode.com/chains/stellar)  | Not available | Available | Available   |
+| [Validation Cloud\*](https://validationcloud.io/)       | [Access](https://app.validationcloud.io/)           | Not available | Available | Available   |
+| [QuickNode\**](https://www.quicknode.com/chains/stellar) | [Access](https://www.quicknode.com/chains/stellar) | Not available | Available | Available   |
 | [NowNodes](https://nownodes.io/nodes/stellar-xlm)       | [Access](https://nownodes.io/nodes/stellar-xlm)     | Available     | Available | Coming soon |
 | [Gateway](https://gateway.fm/)                          | [Access](https://gateway.fm/public-rpc/)            | Not available | Available | Available   |
 
-\*_Please note that Quicknode combines Horizon and RPC in the same endpoint._
+\*_Please note that Validation Cloud provides a websocket (wss) endpoint for
+Testnet and Mainnet._
+\**_Please note that Quicknode combines Horizon and RPC in the same endpoint._
```

Comment:
Add a space above this line, otherwise the markdown will be parsed as if these two notes were meant to be a single line